x-team/xp-ui
# Add LoginButton presentation component

**Release Type:** *Dev Improvements*

Fixes https://x-team-internal.atlassian.net/browse/XP-3296

## Description

This change adds a presentational component - `LoginButton`. It presents a ui/button specifically for the admin login page.

## Checklist

## Related PRs

branch | PR
------ | ------
`XP-3296-use-xp-ui-button` | [XP](https://github.com/x-team/xp/pull/2002)

## Steps to Test or Reproduce

1. Open storybook
1. Search for `LoginButton`
1. Observe basic and fallback story

## Screenshots

![Desktop Recording September 22nd, 2019](https://user-images.githubusercontent.com/15269623/65384111-2ba5b100-dcd3-11e9-88c5-516c7ce2c10b.gif)
